### ISSUE TITLE
Reproduce the results  on HumanEval

### ISSUE THREAD

**Issue by fghccv** [meta-llama/llama3#101]
Can anyone reproduce the results of the Llama3-8B-Instruct on HumanEval（62.2）?
I test it and only got 52.6%(greedy) 
I did not add any prompt and directly used the HumanEval sample as input.

**Comment by ganler** [comment]
61.6 for 8b instruct on HumanEval (https://evalplus.github.io/leaderboard.html) evaluated using EvalPlus (https://github.com/evalplus/evalplus).

**Comment by gracequeen** [comment]
55% for 4-bit quantized 8b-instruct.

**Comment by den-run-ai** [comment]
@fghccv did you use transformers or vllm for inference?

**Comment by cczslp** [comment]
@den-run-ai Yes. I used transformers library and set temperature=0.7 and top_p=0.95 for multinomial sampling. The pass@1 I got under this setting is only 41.46%.
